src/components/PectraUpgrade/Intro/index.tsx

```diff
@@ -0,0 +1,38 @@
+import { Button, Container, Grid, Typography } from '@mui/material'
+import type { BaseBlock } from '@/components/Home/types'
+import css from './styles.module.css'
+import Link from 'next/link'
+import HeroShape from '@/public/images/PectraUpgrade/hero-shape.svg'
+const Intro = ({ text, title, link }: BaseBlock) => {
```

Comment:
Please leave an empty line between the imports and the component function.

```suggestion

const Intro = ({ text, title, link }: BaseBlock) => {
```